spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe 'User Stories' do
+
+  let(:menu)  { Menu.new  }
+  let(:order) { Order.new(menu.options) }
+
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+  it 'customers can see a list of items with prices' do
+    expect(menu.options).to eq({"beef-chuck brisket sub"=>4.5, "pastrami salami sub"=>5.0})
+  end
+
+  # As a customer
+  # So that I can order the meal I want
+  # I would like to be able to select some number of several available dishes
+  it 'allows users to add any amount of dishes to their orders' do
+    menu.my_order.choose('beef-chuck brisket sub', 2)
+    expect(menu.my_order.current).to eq({"beef-chuck brisket sub"=>2})
+  end
+
+  it 'does not allow customers to order things that are not on the menu' do
+    expect{ menu.my_order.choose('something', 3) }.to raise_error'Sorry mate- something is not on the menu!'
+  end
+
+  # As a customer
+  # So that I can verify that my order is correct
+  # I would like to check that the total I have been given matches the sum of the various dishes in my order
+  it 'initialize order with total of 0' do
+    expect(order.total).to eq "Your total is: £0.00."
+  end
+
+  it 'adds up price of each selection and stores it in the @total attribute of order' do
```

Comment:
Metrics/LineLength: Line is too long. [88/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)